pom.xml

```diff
@@ -487,11 +506,23 @@
                                     <pattern>org.slf4j</pattern>
                                     <shadedPattern>${shadeBase}.org.slf4j</shadedPattern>
                                 </relocation>
+                                <relocation>
```

Comment:
Nit: move this up to be directly after the other Jackson dependency (there's no exact ordering to this list, but I tried to group related things together)